nzedb/Desura.php

```diff
@@ -36,7 +36,7 @@ class Desura
 	/**
 	 * @var array
 	 */
-	protected $_postParams = [];
+	private $_postParams = [];
```

Comment:
Why the visibility change? If there is no compelling and immediate reason for making this private it should be left protected so inheriting (now or in the future) classes have access to it.

Reply:
Desura and Greenlight classes should be  actualy dropped, as they do not exist anymore.